### DISCUSSION TITLE
Mean Centering vs Normalize[0,1]

### DISCUSSION THREAD

**Discussion by robcurtis123** [sibylhe/mmm_stan#27] [General]
Hi Sibyl. In the google paper they normalize the variables on a 0 to 1 scale. In your blog post the variables are mean centered. Is there a reason why you made this change? 

To me, it is intuitive to have the intercept be equivalent to when there is no media. But if you mean center the variables, the intercept is where spend is at their average.

**Comment by sibylhe** [comment]
Please see Q4 in #7 

> To me, it is intuitive to have the intercept be equivalent to when there is no media. But if you mean center the variables, the intercept is where spend is at their average.

Can you give an example, what's your x, y, intercept and model formula?

**Reply by robcurtis123** [reply]
Hi. I notice that you divide by the mean when you center your variables.  Would it make more sense to just subtract the mean to mean center? https://www.theanalysisfactor.com/centering-and-standardizing-predictors/

My formula (in simple terms) is sales ~ intercept + B1*media_spend + B2*promo.

**Comment by sibylhe** [comment]
Subtracting mean will result in negative values, because my model is multiplicative, the next step is log1p transformation (i.e., x_new = log(x+1)), I need to avoid negative values.
But your model is additive, you're free to use any kind of normalization (substracting mean, z-score, dividing by mean, etc). There's no concern about negative values.
